work-management/README.md

```diff
@@ -0,0 +1,15 @@
+# Work Management
+We aim to track all work at Tilt in [Clubhouse](https://app.clubhouse.io/windmill) and Google docs, together as the single source of truth, system of record. In particular, this includes future plans and discussions, prioritization and existing work, and past work. This includes product changes, as well as other typical activities requiring collaboration (e.g. planning for a conference).
+
+## User interactions
+In addition to email, Tilters communicate with non-Tilter users via [Kubernetes Slack](https://slack.k8s.io/) in the `#tilt` channel. Users are encouraged to submit feature requests and bugs in [GitHub issues](https://github.com/windmilleng/tilt/issues), where they can subscribe to notifications, discuss ideas with Tilters, and with other users. Open source contributors are also encouraged to participate in the same repo.
+
+## Link GitHub issues and Clubhouse stories together
+All GitHub issues in all [Tilt repos](https://github.com/windmilleng) should have a corresponding Clubhouse story, since all work should be tracked in Clubhouse. (The reverse is not true.) [Private Tilt repos](https://github.com/windmilleng?type=private) should have their respective issue trackers disabled, so that Tilters won't mistakenly create GitHub issues in those cases.
+
+A GitHub issue should only be created in these scenarios:
+- A user creates an issue as a feature request, improvement, or bug.
+- A Tilter creates an issue on a user's behalf (e.g. in response to a Slack conversation).
+- A Tilter intentionally wants to share planned or ongoing work with the community to get feedback. They create a GitHub issue (associated with an already created Clubhouse story).
+
+We are [considering how to to easily link GitHub issues to Clubhouse stories](https://app.clubhouse.io/windmill/story/4692). For now, a GitHub issue's description should contain a Clubhouse story URL. A Clubhouse story that has one or more associated GitHub issues should indicate those GitHub URLs using the [Clubhouse external links feature](https://help.clubhouse.io/hc/en-us/articles/360000272903-Using-the-Zendesk-Integration-and-External-Ticket-Support).
```

Comment:
> For now, a GitHub issue's description should contain a Clubhouse story URL.

Hmm, this seems... potentially weird for users. At minimum let's be sure to call the link something like "ticket in internal issue tracker" or something so users don't get confused when they click on it. But I wonder if there's a way to associate GH --> CH without providing a weird/unclickable experience to users viewing the issue.

Reply:
I'm not aware of any simple way to have GitHub surface some UI that only a signed-in Clubhouse user would see. I updated the content here to say what you said about `internal`.

Alternatively, we could simply _not_ include a link back to Clubhouse. I think that would be annoying for us. But perhaps in this first round of process change, we don't worry about that just yet? What do you think?

Reply:
I don't feel strongly either way--as much as it might be odd for external users, having a GH --> CH link is probably useful, especially since I just checked and you _can't_ find a clubhouse ticket by searching for the text of an attached link (maybe I'll go make a feature request). I'm fine with whatever as long as users don't think they're supposed to be able to click that link / don't get stressed out when they can't.

Reply:
@dbentley : I think you wanted a good experience for users when they are navigating GitHub issues for Tilt. We are proposing something that looks like this. Would this be okay with you?

<img width="754" alt="Screen Shot 2020-02-25 at 5 14 46 PM" src="https://user-images.githubusercontent.com/10860550/75292574-741d4980-57f2-11ea-83be-e2a753a4a4b9.png">

cc @nicks